Form-Controls/index.html

```diff
@@ -10,13 +10,49 @@
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>t-shirt</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+
+    <form>
+      <label for="username">user name</label><br>
+      <input id="username" type="text" minlength="2" maxlength="8" required placeholder="Username that must be between 3 and 8 characters" /><br>
+    
+      <label for="email">Email</label><br>
+      <input id="email" type="email" required /><br>
+  
+      <label for="size">size</label>
+      <select id="size" name="size">
+        <option value="xs" selected>xs</option>
```

Comment:
If XS is selected, what would happen if the user did not make an active choice on this select?